content/videos/challenges/61-fractal-spirograph/index.json

```diff
@@ -0,0 +1,52 @@
+{
+    "title": "Fractal Spirograph",
+    "description": "In this coding challenge, I attempt to create a \u201cFractal Spirograph\u201d with Processing (Java). The visuals are inspired by images and explanation on C. J. Chen\u2019s blog.",
+    "videoNumber": "61",
+    "videoId": "0dwJ-bkJwDI",
+    "date": "2017-02-28",
+    "languages": ["processing", "java"],
+    "topics": [
+        "fractal spirograph",
+        "creative coding",
```

Comment:
I don't think we need a creative coding topic, it's maybe too broad and applies to almost everything?